packages/modular-scripts/react-scripts/config/webpack.config.js

```diff
@@ -634,8 +634,9 @@ module.exports = function (webpackEnv) {
           // See https://github.com/cra-template/pwa/issues/13#issuecomment-722667270
           maximumFileSizeToCacheInBytes: 5 * 1024 * 1024,
         }),
-      // TypeScript type checking
+      // TypeScript type checking for non CI envs
```

Comment:
Can we expand a comment why we're not enabling it for CI environments. Maybe with a link to the GH issue.

Reply:
Not sure which GH issue you're referring to. This is not part of the memory caching issue for ts-jest if that's what you were referring to but I can link the #605 issue in the comment if that is what you meant?

Reply:
Yup - let's just add a link to 605.